src/Icon/Icon.js

```diff
@@ -0,0 +1,115 @@
+// @flow weak
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+
+export const styleSheet = createStyleSheet('Icon', (theme) => {
+  const { palette } = theme;
+  return {
+    accent: {
+      color: palette.accent.A200,
+    },
+    action: {
+      color: palette.action.active,
+    },
+    contrast: {
+      color: palette.getContrastText(palette.primary[500]),
+    },
+    disabled: {
+      color: palette.action.disabled,
+    },
+    error: {
+      color: palette.error[500],
+    },
+    primary: {
+      color: palette.primary[500],
+    },
+  };
+});
+
+/**
+ *
+ * ```jsx
+ * <Icon>account_circle</Icon>
+ * ```
+ */
+const Icon = (props, context) => {
```

Comment:
All our other components are using a named function for declaring the body of the render method.
I'm using the arrow function on the other projects I'm working on. The community seems to embrace it but for some reason we don't. I wish we had @nathanmarks point of view on that.